Formula/boost-python.rb

```diff
@@ -1,8 +1,10 @@
 class BoostPython < Formula
-  desc "C++ library for C++/Python interoperability"
+  desc "C++ library for C++/Python 2 interoperability"
```

Comment:
It's a nit but `Python2` seems more correct than `Python 2`.

Reply:
Done